src/Components/DynamicContent/PromotionBuilder/PromotionBuilder.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropsTypes from 'prop-types'
+
+import NewPromotionTitle from './NewPromotionTitle';
+import PromotionDetails from './PromotionDetails';
+import PromotionToolBar from './PromotionToolBar';
+
+const PromotionBuilder = (props) => {
+    
+    return(<div className="PromotionBuilder">
+        <NewPromotionTitle/>
+        <PromotionDetails  formData={props.formData} onFormChange={props.onFormChange}/>
+        <PromotionToolBar formData={props.formData} selectedPromotionId={props.selectedPromotionId} searchDisplay={props.searchDisplay} reset={props.reset} savePromotion={props.savePromotion}/>
+    </div>)
+}
+
+PromotionBuilder.propTypes = {
+    formData: PropsTypes.object.isRequired, 
```

Comment:
would recommend using `shape` instead of `object`. It provides a more descriptive view of the object, which helps in understanding the code and flow better.